### PR TITLE
fix(html): do not skip custom headers for 3xx responses

### LINKED ISSUE #153: check if redirects also send custom headers
**Description**
from slack:

> I think it works fine if it wasn't 301. let me check..

> Unfortunately the pipeline will not return your response headers with the 301 as it stands today.

### PR DESCRIPTION
**Pull request by trieloff**
fixes #153

### CODE REVIEW

File: src/html-pipe.js [hunk 1]
```diff
@@ -98,6 +98,9 @@ export async function htmlPipe(state, req) {
       const level = res.status >= 500 ? 'error' : 'info';
       log[level](`pipeline status: ${res.status} ${res.error}`);
       res.headers.set('x-error', cleanupHeaderValue(res.error));
+      if (res.status <= 400) {
```
Comment (tripodsan) [inline src/html-pipe.js:101]: why `<= 400`  and not `< 400` or `< 500` ?

Comment (dylandepass) [reply src/html-pipe.js:101]: I would think we should return the response headers for a 404 as well. So feels like it should be `< 500`.

Comment (trieloff) [reply src/html-pipe.js:101]:
```suggestion
      if (res.status < 500) {
```

**Comment by trieloff** [comment]
Now for all 3xx and all 4xx (not just 3xx + 400)

**Review by dylandepass — APPROVED** [review]
🚀

**Pull request merged**